### PR TITLE
Add mobilenet V1 1.0 224 fine tune script 

### PR DESCRIPTION
**Pull request by chungyeh**
I signed it!

### CODE REVIEW

**Comment by AndreaPisoni** [comment]
Is this script correct? I do not see any reference to "AuxLogits" in the mobilenet architecture file, (which is different from inception, where AuxLogits is present...) so probably the layers to be chosen would need to be different?

**Comment by haoxi911** [comment]
I didn't see 'AuxLogits' in MobilenetV1, but 'Logits' did exist, maybe we just want to remove 'AuxLogits' from the script?

Also, I am not using 'flower' and instead a own dataset, is there any additional steps required?

**Comment by haoxi911** [comment]
@AndreaPisoni @chungyeh @bmabey 

I can train the MobilenetV1 checkpoint files using my own dataset, however the checkpoint file can't be frozen due to the following error:

> InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [1,1,1024,1001] rhs shape= [1,1,1024,37]
> 	 [[Node: save/Assign_136 = Assign[T=DT_FLOAT, _class=["loc:@MobilenetV1/Logits/Conv2d_1c_1x1/weights"], use_locking=true, validate_shape=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](MobilenetV1/Logits/Conv2d_1c_1x1/weights, save/RestoreV2_136)]]

I believe it is the problem of `--checkpoint_exclude_scopes` or `--trainable_scopes` options I passed when finetune the model:

> (tensorflow) ubuntu@haoxi911:~/tensorflow/models/research/slim$ python train_image_classifier.py --train_dir=/tmp/oxford-iiit-pet/mobile_v1_1.0_224_finetune --dataset_dir=/tmp/oxford-iiit-pet --dataset_name=oxford_iiit_pets --dataset_split_name=train --model_name=mobilenet_v1 --checkpoint_path=/tmp/mobilenet_v1_2018_02_22/mobilenet_v1_1.0_224/mobilenet_v1_1.0_224.ckpt --checkpoint_exclude_scopes=MobilenetV1/Logits --trainable_scopes=MobilenetV1/Logits --clone_on_cpu=True

I also tried these options but got same errors:

> --checkpoint_exclude_scopes=MobilenetV1/Logits/Conv2d_1c_1x1/weights
> --trainable_scopes=MobilenetV1/Logits/Conv2d_1c_1x1/weights

I have posted the same question to SO [here](https://stackoverflow.com/questions/49062540/how-do-i-finetune-mobilenetv1-with-my-own-dataset), it would be great if you can take a look at the problem.
